flag_engine/environments/schemas.py

```diff
@@ -28,11 +28,8 @@ class Meta:
 
 
 class WebhookSchema(LoadToModelSchema):
-    enabled = fields.Bool()
```

Comment:
Hmm, enabled  was never part of webhook Django model?

Reply:
Well, it's part of the `Webhook` class in `environments.models` but not part of the `WebhookConfiguration` model that we should be using.